README.md

```diff
@@ -53,11 +58,11 @@ Card (secondary, outlined, color, isClickable, noPadding)
 Checkbox (label, checked, disabled, uppercase)
 
 Datatable (label) [
-	Datatable.Data, Datatable.Data.Row (clickable), Datatable.Data.Row.Item, Datatable.Header, Datatable.Header.Item
+Datatable.Data, Datatable.Data.Row (clickable), Datatable.Data.Row.Item, Datatable.Header, Datatable.Header.Item
 ]
 
 Dialog [
-	Dialog.Alert (open, title, defaultAction), Dialog.Simple(open, title)
+Dialog.Alert (buttons: [{label: 'cancel', action: 'cancel',class: 'mdc-dialog__button'},{label: 'discard', action: 'discard',class: 'mdc-dialog__button'}], open: false, title: '', defaultAction: 'discard'), Dialog.Simple(items: [], open: false, title: '')
```

Comment:
This seems to be (one of?) the only place(s) showing values for the properties. Were the values added intentionally?

Reply:
yes, I originally did not include values. If you think it is a bad idea we can remove them. I have only added them to this elements so far.

Reply:
I think it's a little confusing, but perhaps that's just the formatting

Reply:
I have made it more readable now.